server/models/profileModel.js

```diff
@@ -1,9 +1,6 @@
 const mongoose = require('mongoose');
 
 const profileSchema = {
-    userID:{
```

Comment:
This is okay for now. Let's just keep in mind that we do want the profile to be connected to a user though so later when auth is complete we'll need to reference a userId from the user model.